grails-app/migrations/views/order-summary-view.sql

```diff
@@ -55,74 +51,65 @@ CREATE OR REPLACE VIEW order_receipt_status AS
           AND shipment.current_status = 'RECEIVED' OR shipment.current_status = 'PARTIALLY_RECEIVED'
         GROUP BY `order`.id, `order`.order_number, order_item.id, shipment.id
     )
-AS order_receipt_status;
+AS order_receipt_status GROUP BY order_item_id;
 
-CREATE OR REPLACE VIEW order_payment_status_from_shipments AS
+CREATE OR REPLACE VIEW order_item_payment_status AS
     SELECT
         order_id,
         order_number,
         order_item_id,
-        quantity_ordered,
-        shipment_item_quantity_invoiced,
-        order_status,
-        (quantity_ordered > 0)                                AS shipment_payment_ordered,
-        (shipment_item_quantity_invoiced >= quantity_ordered) AS shipment_item_invoiced,
-        invoice_submitted
+        SUM(quantity_ordered)  				AS quantity_ordered,
+        IFNULL(SUM(quantity_invoiced), 0) 	AS quantity_invoiced
```

Comment:
I wonder if it would be a good iea to use the IFNULL convention for all of these SUMs.

Reply:
I added it in some other PR, because it turned out it was a good idea